pagopa/pn-frontend
# PN-1814 - add FAQ page to landing site

## Short description
Add a /faq page in the landing site.

## List of changes proposed in this pull request
- Defined the data structures for this page, which is not like the currently existing pages.
- Defined the contents in Italian (only language for which the text were provided).
- Defined a new `getFaqData` function in `api/index.ts` that returns always the Italian version (as it is the only one available).
- Added the code to render the page.
- Added the entry in the navigation bar.
- Added three new env variables: `NEXT_PUBLIC_PIATTAFORMA_NOTIFICHE_PF_URL`, `NEXT_PUBLIC_PIATTAFORMA_NOTIFICHE_PG_URL` and `NEXT_PUBLIC_PERFEZIONAMENTO_PATH`. Here are some suitable values: 
```
NEXT_PUBLIC_PIATTAFORMA_NOTIFICHE_PF_URL=https://cittadini.notifichedigitali.it
NEXT_PUBLIC_PIATTAFORMA_NOTIFICHE_PG_URL=https://imprese.notifichedigitali.it
NEXT_PUBLIC_PERFEZIONAMENTO_PATH=/perfezionamento
```

## How to test
Launch pn-landing-webapp, point to `localhost:3000/faq`, see what happens.  
Each section is linked to an id so that the addresses `localhost:3000/faq#<id>` should also work. This is needed to include links to specific FAQ sections in the PA / PF / PG webapps.
